core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2135,8 +2177,8 @@ private void mountAndJournal(LockedInodePath inodePath, AlluxioURI ufsPath, Moun
       throw new InvalidPathException(
           ExceptionMessage.MOUNT_POINT_ALREADY_EXISTS.getMessage(inodePath.getUri()));
     }
-
-    mountInternal(inodePath, ufsPath, false /* not replayed */, options);
+    long mountId = IdUtils.createMountId();
```

Comment:
IDs can be created, or derived from journal. So I leave the methods in `MountTable` to take mount id as parameters so journal replay is easier.